easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-5.1.1-2023a.yml

```diff
@@ -10,3 +10,4 @@ easyconfigs:
         # See https://github.com/easybuilders/easybuild-easyconfigs/pull/22932
         from-commit: 3211d34eb16ff31b8de3dfef55ecaaf1ec205c6f
   - MrBayes-3.2.7-gompi-2023a.eb
+  - code-server-4.97.2.eb
```

Comment:
This is using the SYSTEM toolchain, so could you please move it to the corresponding easystack file (https://github.com/EESSI/software-layer/blob/main/easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-5.1.1-001-system.yml)?